indexing.sql

```diff
@@ -0,0 +1,37 @@
+CREATE USER indexed_cars_user;
+CREATE DATABASE indexed_cars OWNER indexed_cars_user;
+\c 'indexed_cars';
+
+SELECT COUNT(*) FROM car_models; 
+
+SELECT DISTINCT make_title FROM car_models WHERE make_code ~ 'LAM';
+-- 1 result
+SELECT DISTINCT model_title FROM car_models WHERE make_code ~ 'NISSAN' AND model_code ~ 'GT-R';
+-- 1 result
+SELECT DISTINCT COUNT(model_title) FROM car_models WHERE make_code ~ 'LAM';
+-- 1360 rows
+SELECT DISTINCT COUNT(*) FROM car_models WHERE year BETWEEN 2010 AND 2015;
+-- 78840 rows
+SELECT DISTINCT COUNT(*) FROM car_models WHERE year = 2010;
+-- 13140 rows
+
+CREATE INDEX title_car 
+  ON car_models (make_title);
+
+SELECT DISTINCT make_title FROM car_models WHERE make_code ~ 'LAM';
+DROP INDEX title_car;
```

Comment:
You don't need to drop any of these indexes, actually you should be making all three indexes at the same time, then run the same five queries again after you've created the three indexes.